Assets/Scripts/Game/Items/DaggerfallUnityItemMCP.cs

```diff
@@ -0,0 +1,89 @@
+// Source Code:     https://github.com/Interkarma/daggerfall-unity
+// Original Author: Hazelnut
+
+using System;
+using DaggerfallWorkshop.Utility;
+
+namespace DaggerfallWorkshop.Game.Items
+{
+    public partial class DaggerfallUnityItem : IMacroContextProvider
+    {
+        public MacroDataSource GetMacroDataSource()
+        {
+            return new ItemMacroDataSource(this);
+        }
+
+        /// <summary>
+        /// MacroDataSource context sensitive methods for items in Daggerfall Unity.
+        /// </summary>
+        private class ItemMacroDataSource : MacroDataSource
+        {
+            private string[] conditions = new string[] { "Broken", "Useless", "Battered", "Worn", "Used", "Slightly Used", "Almost New", "New" };
+            private int[] conditionThresholds = new int[] {1, 5, 15, 40, 60, 75, 91, 101};
+
+            private DaggerfallUnityItem parent;
+            public ItemMacroDataSource(DaggerfallUnityItem item)
+            {
+                this.parent = item;
+            }
+
+            public override string ItemName()
+            {
+                return parent.shortName;
+            }
+
+            public override string Worth()
+            {
+                return parent.value.ToString();
+            }
+
+            public override string Material()
+            {   // %mat
+                switch (parent.itemGroup)
+                {
+                    case ItemGroups.Armor:
+                        return ((ArmorMaterialTypes)parent.nativeMaterialValue).ToString();
+                    case ItemGroups.Weapons:
+                        return ((WeaponMaterialTypes)parent.nativeMaterialValue).ToString();
+                    default:
+                        return base.Material();
+                }
+            }
+
+            public override string Condition()
+            {   // %qua
+                if (parent.maxCondition > 0 && parent.currentCondition <= parent.maxCondition)
+                {
+                    int conditionPercentage = 100 * parent.currentCondition / parent.maxCondition;
+                    int i = 0;
+                    while (conditionPercentage > conditionThresholds[i])
+                        i++;
+                    return conditions[i];
+                }
+                else
+                    return parent.currentCondition.ToString();
+            }
+
+            public override string Weight()
+            {   // %kg
+                return parent.weightInKg.ToString();
+            }
+
+            public override string WeaponDamage()
+            {   // %wdm
+                int matMod = parent.GetWeaponMaterialModifier();
+                return String.Format("{0} - {1}", parent.GetBaseDamageMin() + matMod, parent.GetBaseDamageMax() + matMod);
+            }
+
+            // Armour mod not correct yet.. need to refactor DaggerfallEntity.UpdateEquippedArmorValues()
```

Comment:
Looks like the armor value displayed by classic in the popup is half the value shown floating around your character on the inventory screen (which is the value GetMaterialArmorValue() returns). It's rounded down in case it doesn't divide evenly, like with leather, which is "3" and so shows as 1.

Reply:
This does not seem to be the case for shields, which show the same value as gets added to the floating numbers around the character.
This seems like a slip up in the original game, so I think we should just show the values as you have them now.

Reply:
Okay, I assumed DFU was mistaken. :-)

You're the oracle for this stuff as far as I'm concerned!